server/command.go

```diff
@@ -78,7 +78,8 @@ const (
 	projectNotFoundError   = "404 {message: 404 Project Not Found}"
 	projectNotFoundMessage = "Unable to find project with namespace: "
 
-	invalidSubscribeSubCommand = "Invalid subscribe command. Available commands are add, delete, and list"
+	invalidSubscribeSubCommand          = "Invalid subscribe command. Available commands are add, delete, and list"
+	missingOgOrRepoFromSubscribeCommand = "Please provide the owner[/repo] [features]"
```

Comment:
```suggestion
	missingOrgOrRepoFromSubscribeCommand = "Please provide the owner[/repo]"
```
Can skip [features], as that is optional